src/api/utils/time-utils.ts

```
@@ -0,0 +1,23 @@
+    y: 365 * 24 * 60 * 60 * 1000,
+  };
+
+  public static parseRelativeTime(value: string): number {
+    const match = value.match(/^([1-9]\d*)(min|h|d|w|m|y)$/);
+    if (!match) {
+      throw new Error("Invalid relative time format");
+    }
+
+    const amount = Number(match[1]);
+    const unit = match[2];
+    const multiplier = this.unitMap[unit];
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)